addons/l10n_fi_rapko/data/account_chart_template_pre_data.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="account.account.template" id="a19190">
+        <field name="code">19190</field>
+        <field name="user_type_id" ref="account.data_account_type_receivable"/>
+        <field name="reconcile" eval="True"/>
+        <field name="name">Rahansiirrot/ v&#228;litilit/ t&#228;sm&#228;ytykset</field>
+    </record>
+
+    <!-- Account Chart Template -->
+    <record id="chart_template_fi_rapko" model="account.chart.template">
```

Comment:
This is the list of languages in which the chart of account will be translate, see https://github.com/odoo/odoo/blob/10.0/addons/l10n_ch/data/l10n_ch_chart_data.xml#L21 for instance